invenio_search/cli.py

```diff
@@ -63,16 +63,26 @@ def check():
 
 @index.command()
 @click.option('--force', is_flag=True, default=False)
+@click.option(
+    '-i',
+    '--index_name',
```

Comment:
Other commands e.g. `--yes-i-know` are divided with dash rather than underscore. In addition, if we want to add 2 indexes we have to do:

```
--index-name xwy --index-name abc
```

So it's okay to call it `index-name` rather than `index-nameS`. :)